.github/workflows/percy.yml

```diff
@@ -33,9 +34,13 @@ jobs:
       - name: Install Dependencies
         run: npm install
 
+      - name: Install Percy
+        run: npm install --save-dev @percy/cli @percy/selenium-webdriver
```

Comment:
Why not include these in the project's `package.json`? Then they should be installed on the `Install Dependencies` step also.